clang/test/SemaSYCL/sycl-no-rdc.cpp

```diff
@@ -0,0 +1,25 @@
+// RUN: %clang_cc1 -fsycl-is-device -verify -fsyntax-only -fsycl-allow-func-ptr -fno-gpu-rdc %s
```

Comment:
Please add a comment here about what this test checks for.

Reply:
done thanks